### DISCUSSION TITLE
Where to put rkflashtool.exe?

### DISCUSSION THREAD

**Discussion by bh-hs** [ntherning/StorytelReaderMods#5] [Q&A] [answered]
Hello, thank you for creating this guide! I'm currently stuck on this step:

_For the remainder of this doc it is assumed that rkflashtool.exe is in the current directory or has been installed to a directory listed in the Windows %PATH% environment variable._

I'm trying to run rkflashtool.exe but it immediatly closes. What directory I should put it in?

Thanks!

**Comment by ntherning** [comment, marked as answer]
You need to run all commands in a command prompt, i.e. `cmd.exe`. It sounds like you tried to double-click the `rkflashtool.exe` file?

**Comment by bh-hs** [comment]
That worked, thank you!
